OHDSI

```diff
@@ -0,0 +1 @@
+https://athena.ohdsi.org/vocabulary/list
```

Comment:
Hello @sunithae21 , thank you for contributing to Athena. Can you, please,  provide more details? What exactly fix this PR, and why did you do this way?